app/controllers/reservations_controller.rb

```diff
@@ -12,7 +15,7 @@ def create
     @reservation.user = current_user
     @reservation.is_confirmed = false
     if @reservation.save
-      redirect_to space_path(@space)
+      redirect_to space_path(@space) # TODO: reservations_path
```

Comment:
you mean to create an index of my_reservations? If so, agree; makes sense

Reply:
✅